Extension/package.nls.json

```diff
@@ -1025,15 +1028,15 @@
     },
     "c_cpp.walkthrough.create.cpp.file.altText": "Open a C++ file or a folder with a C++ project.",
     "c_cpp.walkthrough.command.prompt.title": {
-        "message": "Launch from the Developer Command Prompt for VS",
+        "message": "Apply the Visual Studio Developer Environment",
```

Comment:
I think `Developer Environment` should be lower cased, right? Why is it upper cased? Usually it would only be upper cased if it was a product name, but I don't think it is. Using upper case might confuse the translators too. It's not upper cased in `c_cpp.configuration.persistVSDeveloperEnvironment.markdownDescription`.

Reply:
The text for all commands should be using Title Casing.  Settings descriptions use sentence casing.

Reply:
@bobbrow I don't understand -- `c_cpp.walkthrough.command.prompt.title` is just an arbitrary string right? You could change "command" in the string to anything. It's not actually a "command" that requires title casing. It's just a label for a string we show users in a button. Are you saying all button text should be Title cased?

Reply:
Also, it uses capital Developer Environment in the string `"{0} requires the Visual Studio Developer Environment.`

Reply:
Oh, but it does look like the recommended style for button text is to use title case, so that seems fine to me, but the other cases of "Developer Environment" that aren't in the button text should be non-title cased.

Reply:
> Also, it uses capital Developer Environment in the string `"{0} requires the Visual Studio Developer Environment.` and other messages that aren't part of a "command".

This one should already be fixed in the latest update.

Reply:
> @bobbrow I don't understand -- `c_cpp.walkthrough.command.prompt.title` is just an arbitrary string right? You could change "command" in the string to anything. It's not actually a "command" that requires title casing. It's just a label for a string we show users in a button. Are you saying all button text should be Title cased?

I'm not sure where I went wrong here. For some reason I thought you were referring to the "persist" setting. In any case, I looked through the other walkthrough page titles and it looks like we're not totally consistent on the capitalization. The VS Code documentation pages also are inconsistent, but I do see enough sentence casing in the page titles that I'm fine if we standardize to that. I'll update the other strings in this PR as well.